libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/LaneIconProcessor.kt

```diff
@@ -48,7 +48,8 @@ internal class LaneIconHelper {
             R.drawable.mapbox_ic_lane_sharp_right_straight_straight_only,
     )
 
-    fun retrieveLaneToDraw(laneIndicator: LaneIndicator, activeDirection: String?): Int? {
+    @DrawableRes
+    internal fun drawable(laneIndicator: LaneIndicator, activeDirection: String?): Int? {
```

Comment:
We can remove `internal` to maintain consistency with other `Processors`. 
Also let's change the function name to `getDrawableFrom` as `drawable` is not intuitive

Reply:
agree 👍  
IMO prefer `map` over `get` for these `mapDrawableFrom`. will do get for now to be consistent